src/index.html

```diff
@@ -7,7 +7,58 @@
     <link rel="stylesheet" href="./index.scss">
   </head>
   <body>
-    <h1>Hello, email client!</h1>
+    <div class="wrapper">
+      <div class="email-client">
+        <div class="email-list-container">
+          <h2 class="email-header">Inbox</h2>
+          <ul class="email-list">
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email1 subject</p>
+              <p>Email snippet</p>
+            </li>
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email2 subject</p>
+              <p>Email snippet</p>
+            </li>
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email3 subject</p>
+              <p>Email snippet</p>
+            </li>
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email4 subject</p>
+              <p>Email snippet</p>
+            </li>
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email5 subject</p>
+              <p>Email snippet</p>
+            </li>
+          </ul>
+        </div>
+        <div class="email-show-container">
```

Comment:
...and make this a `main` tag (semantics again 😃)

Reply:
👍